src/strategies/latin_hypercube.jl

```diff
@@ -0,0 +1,101 @@
+"""
+LatinHypercube(nGenerations = 1, popSize = 100, nTournament = 2,
+                pTournament = 0.8. interSampleWeight = 1.0,
+                ae_power = 2, periodic_ae = false)
+
+Instantiate  grid-based hyperparameter tuning strategy using the library
+LatinHypercubeSampling.jl. The optimised Latin Hypercube sampling plan is
+created using a genetic based optimization algorithm based on the inverse of the
+Audze-Eglais function.
+The optimization is run for nGenerations. The population size, number of samples
+selected, probability of selection, the inter sample weight of sampling and the
+norm can be choosen. There is also the possibility of using a periodic version
+of the Audze-Eglais which reduces clustering along the boundaries of the
+sampling plan. This feature can be specified by the periodic_ae variable.
+
```

Comment:
To enable this feature set `periodic_ae=true`. 
Wrap all variable references in doc-strings in back ticks `.